src/js/view/controlbar.js

```diff
@@ -307,6 +312,15 @@ define([
         onCastActive : function(model, val) {
             utils.toggleClass(this.elements.cast.element(), 'jw-off', !val);
         },
+        onAirplayAvailable : function(model, val) {
+            this.elements.airplay.toggle(val);
+        },
+        onAirplayActive : function(model, val) {
+            utils.toggleClass(this.elements.airplay.element(), 'jw-off', !val);
+            if (this.elements.mute) {
+                this.elements.mute.toggle(!val);
```

Comment:
Hey @robwalch, I would expect this to hide the volume button, but it doesn't seem to be working. Any ideas?